src/Publisher.js

```diff
@@ -23,12 +23,60 @@ class Publisher {
     return channel.assertExchange(this.exchange, 'fanout', {durable: true})
   }
 
+  /**
+   * @param {String} action
+   * @param {*} data
+   * @param {String} correlationId
+   * @param {Number} timeOut
+   * @return {Promise}
+   * */
+  sendAction (action, data, correlationId, timeOut = null) {
```

Comment:
Same thing happend in RPCClient. You can find more details at `RPCClient@callAction`

Point is. What about something like this:
```javascript
sendAction (action, data, correlationId, timeOut = null) {
    return this.send({ action, data }, correlationId, timeOut)
}
```

Reply:
fix